src/scss/app.scss

```diff
@@ -117,6 +117,11 @@ a {
   transition-duration: 150ms;
   transition-property: opacity;
 
+  @media screen and (max-width: 1080px) {
+    max-height: 300px;
```

Comment:
I was just looking around and saw this PR, I don't specifically know what problem you trying to solve but I think a fixed height like `300px` might now be the best solution, maybe using the view height property be the best thing to do like this:
```suggestion
    max-height: 1vh;
```

Reply:
Ya, vh is better suited, thanks